.buildkite/pipeline-nightly-e2e-tests.yml

```diff
@@ -0,0 +1,348 @@
+agents:
+  memory: "4G"
+
+steps:
+
+  - group: "aks"
+    steps:
+
+      - label: "aks"
```

Comment:
```suggestion
      - label: "run e2e tests"
```

I think generally, I'm asking if the labels can be a bit more descriptive?

Reply:
I chose them short, perhaps a bit too much. In the UI, the groups of this pipeline appear after the `e2e-tests` step that upload the pipeine. I thought it could be enough.

I admit that this one seems particularly useless. Repeating 'aks' as the name of the group brings nothing. Ideally, this is the version of aks that I would like to present here. But currently this version is maintained in the [hack/deployer/config/plans.yml](https://github.com/elastic/cloud-on-k8s/blob/main/hack/deployer/config/plans.yml#L48) file and I want to avoid duplicating the version information in two different places and I did not dare to change the existing behaviour.

![2022-10-04-103753_1287x256_scrot](https://user-images.githubusercontent.com/582883/193774080-c9f96d50-93d2-449d-9fc2-ad5c760b7c66.png)